torch/_meta_registrations.py

```diff
@@ -465,6 +489,110 @@ def linalg_cholesky_ex(A: Tensor, upper: bool = False, check_errors: bool = Fals
     return L, infos
 
 
+def householder_product_out_helper(
+    input: Tensor,
+    tau: Tensor,
+    result: Tensor,
+) -> Tensor:
+    assert input.ndim >= 2
+    assert input.size(-2) >= input.size(-1)
+    assert input.size(-1) >= tau.size(-1)
+
+    assert input.dtype == tau.dtype
+    assert input.device == tau.device
+
+    assert result.dtype == input.dtype
+    assert result.device == input.device
+
+    # if result has no elements we can modify it
+    if result.numel() == 0:
+        result = _maybe_resize_out(result, input.mT.shape)
+        result.transpose_(-2, -1)
+
+    # result tensor must be in batched column major order (Fortran contiguous)
+    assert result.mT.is_contiguous()
+    assert result.shape == input.shape
+
+    result.copy_(input)
+
+    return result
```

Comment:
Same name as in core. I don't call the stub here because it's a meta. But I wonder if I should make tau contiguous as in core, which I've also skipped.

Reply:
Remove this one and simply use the `out_wrapper` decorator.